### PR TITLE
fix: UserService.Get change params username to accountId

### PR DESCRIPTION
**Pull request by clementlecorre**
https://github.com/andygrunwald/go-jira/issues/256

### CODE REVIEW

**Comment by ghostsquad** [comment]
@clementlecorre this looks reasonable, however I'm a bit surprised if this has never worked. Or was this an API change on Jira's end?

**Comment by ghostsquad** [comment]
I see now:
https://developer.atlassian.com/cloud/jira/platform/deprecation-notice-user-privacy-api-migration-guide/

**Comment by ghostsquad** [comment]
@clementlecorre unfortunately, not everyone is going to be running Jira Cloud, or it's possible they haven't updated, and I think this would be a breaking change in that case.

Would you mind instead adding a new method such as `UserService.GetByAccountID` and comment that `Get` will may not work and will be deprecated based on the upstream documentation?

**Comment by clementlecorre** [comment]
Indeed it worked before because I use it regularly.

Indeed it worked before because I use it regularly.
Here is a fix to use the deprecated method if the accountID is not found. The default method is the new (GetByAccountID).

File: issue.go [hunk 1]
```diff
@@ -1236,9 +1237,17 @@ func (s *IssueService) GetWatchers(issueID string) (*[]User, *Response, error) {
 	result := []User{}
 	user := new(User)
 	for _, watcher := range watches.Watchers {
-		user, resp, err = s.client.User.Get(watcher.Name)
-		if err != nil {
-			return nil, resp, NewJiraError(resp, err)
+		if watcher.AccountID != "" {
+			// try fallback deprecated method
```
Comment (ghostsquad) [inline issue.go]: I think this comment is in the wrong place, since `GetByAccountID` is not the deprecated method.

Comment (clementlecorre) [reply issue.go]: Fixed

**Comment by ghostsquad** [comment]
Can you rebase and squash these commits?

**Pull request merged**